src/test/core/get-tracked-source-files.test.ts

```diff
@@ -71,9 +79,19 @@ describe('getTrackedSourceFiles', () => {
       '.mjs',
       '.cjs',
       '.jsx',
-      '.tsx'
+      '.tsx',
+      '.html',
+      '.htm'
     ])
 
     expect(sourceFiles.map((file) => file.fileName)).toStrictEqual([])
   })
+
+  it('correctly parses raw html file into AST', async () => {
+    const root = new Fixture('projects/web-component-project')
+
+    const parsedHtmlFiles = await getTrackedSourceFiles(process.cwd(), root.path, logger, ['.html'])
+
+    expect(parsedHtmlFiles[0]).toMatchSnapshot()
```

Comment:
Like we had been talking about, this is a huge file. So, since we are already implicitly testing if an HTML file is parsable into AST in other tests (by walking through them), let's just check if we are able to get the tracked HTML file in that project directory.